### PR TITLE
Config scss file

### PR DESCRIPTION
**Pull request by Winchestery**
`_variables.scss` file added:
variables
maps
mixins
custom styles setted
`@import '_fonts'`

`'_fonts.scss' `file  added:
`@import` google fonts

`index.scss` changed:
`@import 'src/style/_variables/scss'`

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>WHAT</title>
+    <link href="https://fonts.googleapis.com/css2?family=Libre+Baskerville:ital,wght@0,400;0,700;1,400&family=Libre+Franklin:ital,wght@0,200;0,300;0,400;0,500;1,200;1,300;1,400;1,500&display=swap" rel="stylesheet">
```
Comment (so2niko) [inline src/index.html:8]: Why we connect fonts here? Can we connect it in CSS?

**Review by so2niko — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (Winchestery) [reply src/index.html:8]: Do you mean @import  link in css file 
or @font-face CSS-rule with downloading the fonts?

Comment (sarhan-azizov) [reply src/index.html:8]: `@import`

ref: https://stackoverflow.com/questions/14676613/how-to-import-google-web-font-in-css-file

**Review by sarhan-azizov — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (Winchestery) [reply src/index.html]: Done

**Review by PavelKundenko — APPROVED** [review]

**Review by MozdolevskyiMaksym — APPROVED** [review]

**Review by sarhan-azizov — APPROVED** [review]

**Pull request merged**